example/index.html

```diff
@@ -181,6 +181,30 @@
       tree.select(this.value)
     })
 
+    setTimeout(function () {
+      let nodes = []
+
+      for (var i = 0; i < 1500; i++) {
+        nodes.push(i + 15000)
+      }
+
+      let all = nodes.map(n => {
+        let parentId = 1002
+        let node = {
+          id: n,
+          label: 'New node ' + n,
+          parentId,
+          color: 'gray',
+          itemType: 'objective'
+        }
+        return {
+          data: node,
+          parent: parentId
+        }
+      })
+      tree.addAll(all)
+    }, 3000)
```

Comment:
This block won't hit master, but wanted to show what I was doing to reproduce the bug.